docs/joss/paper.md

```diff
@@ -0,0 +1,217 @@
+---
+title: 'SpeedyWeather.jl: Reinventing atmospheric general circulation models towards interactivity and extensibility'
+
+tags:
+  - Julia
+  - weather
+  - climate
+  - general circulation model
+  - spectral
+  - spherical harmonic transform
+
+authors:
+  - name: Milan Klöwer
+    orcid: 0000-0002-3920-4356
+    email: milank@mit.edu
+    corresponding: true # (This is how to denote the corresponding author)
+    affiliation: "1, 2" # (Multiple affiliations must be quoted)
+
+  - name: Maximilian Gelbrecht
+    orcid: 0000-0002-0729-6671
+    affiliation: "3,4"
+
+  - name: Daisuke Hotta
+    orcid: 0000-0003-2287-0608
+    affiliation: "5,6"
+
+  - name: Justin Willmert
+    orcid: 0000-0002-6452-4693
+    affiliation: 7
+
+  - name: Simone Silvestri
+    affiliation: 1
+
+  - name: Gregory L Wagner
+    orcid: 0000-0001-5317-2445
+    affiliation: 1
+
+  - name: Alistair White
+  - orcid: 0000-0003-3377-6852
+    affiliation: "3,4"
+
+  - name: Sam Hatfield
+    orcid: 0000-0001-7235-6450
+    affiliation: 6
+
+  - name: Tom Kimpson
+    orcid: 0000-0002-6542-6032
+    affiliation: "2,8"
+
+  - name: Navid C Constantinou
+    orcid: 0000-0002-8149-4094
+    affiliation: 9
+
+  - name: Chris Hill
+    affiliation: 1
+    
+affiliations:
+ - name: Massachusetts Institute of Technology, Cambridge, MA, USA
+   index: 1
+ - name: University of Oxford, UK
+   index: 2
+ - name: Technical University of Munich, Germany
+   index: 3
+ - name: Potsdam Institute for Climate Impact Research, Germany
+   index: 4
+ - name: Japan Meteorological Agency, Tsukuba, Japan
+   index: 5
+ - name: European Centre for Medium-Range Weather Forecasts, Reading, UK
+   index: 6
+ - name: University of Minnesota, Minneapolis, MN, USA
+   index: 7
+ - name: University of Melbourne, Australia
+   index: 8
+ - name: Australian National University, Canberra, Australia
+   index: 9
+   
+date: 14 September 2023
+bibliography: paper.bib
+
+---
+
+
+# Summary
+
+SpeedyWeather.jl is a library to simulate and analyze the global atmospheric
+circulation on the sphere. It implements several 2D and 3D
+models solved with spherical harmonics:
```

Comment:
```suggestion
circulation on the sphere. It implements several 2D and 3D
spectral models solving different sets of equations:
```
spherical harmonics is only a 2D discretization method, not 3D, for this reason (and the fast that SpeedyWeather is more than spherical harmonics) I would not emphasize SPH here but when you talking about numerics, maybe where you introduce the modules in the second paragraph